pkg/server/server_test.go

```diff
@@ -0,0 +1,25 @@
+package server
+
+import (
+    "net/http"
+    "net/http/httptest"
+    "testing"
+
+    "github.com/stretchr/testify/assert"
+    "github.com/stretchr/testify/require"
+)
+
+func TestNew(t *testing.T) {
+    srv := New()
+
+    t.Run("serves registered endpoint", func(tt *testing.T) {
+        w := httptest.NewRecorder()
+        req, err := http.NewRequest("GET", "/", nil)
+        require.Nil(tt, err, "unexpected error when making new request")
```

Comment:
FYI: I _think_ there's a `require.NoErr` that's effectively the same as `Nil`; I like it because it feels more descriptive about what I'm trying to test/assert. No need to change this, just another way to spell it.